src/aws_extension.cpp

```diff
@@ -7,12 +7,19 @@
 #include <duckdb/parser/parsed_data/create_scalar_function_info.hpp>
 #include <aws/core/Aws.h>
 #include <aws/core/auth/AWSCredentialsProviderChain.h>
-#include <iostream>
+#include <aws/core/client/ClientConfiguration.h>
 
 namespace duckdb {
 
+struct SetCredentialsResult {
+	string set_access_key_id;
```

Comment:
can we initialize them with an empty string explicitly?
Since we create `SetCredentialsResult ret;` I'm afraid they could be left uninitialized

Though I'm not sure if that's actually an issue in c++ or only in c.

Reply:
the default constructor of string should initialize to an empty string